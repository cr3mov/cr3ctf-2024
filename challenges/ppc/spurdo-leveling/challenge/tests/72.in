100919758076402649 991818847865555930
545043 36302
621836 37367
