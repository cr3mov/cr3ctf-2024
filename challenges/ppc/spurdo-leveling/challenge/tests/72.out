784917589680881875 1541004839832 1339147975826
